Main/ClientApp/src/Components/AdminView.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { useState, useEffect } from "react";
+import UsersStore from "../Stores/UsersStore";
+import ConfigurationStore from "../Stores/ConfigurationStore";
+import {
+  Grid,
+  Card,
+  CardHeader,
+  CardActions,
+  Button,
+  IconButton,
+  Divider,
+  TableContainer,
+  Table,
+  TableHead,
+  TableRow,
+  TableCell,
+  TableBody,
+  Paper,
+} from "@material-ui/core";
+import { Delete } from "@material-ui/icons";
+import ChangeMaxCollectionsDialog from "./Dialogs/ChangeMaxCollectionsDialog";
+import ChangeMaxElementsDialog from "./Dialogs/ChangeMaxElementsDialog";
+import DeleteUserDialog from "./Dialogs/DeleteUserDialog";
+
+export default function AdminView() {
+  const [users, setUsers] = useState(UsersStore.getUsers());
+  const [configuration, setConfiguration] = useState(
+    ConfigurationStore.getConfiguration()
+  );
+  let maxElements = configuration.get("MaxElements");
```

Comment:
You see here why using dictionaries is a bad idea. With a proper class you use properties to access the values instead of get(<string>). It benefits with code completion and no way to make a typo